jobs/config.json

```diff
@@ -199,6 +219,14 @@
   ]
 },
 
+"ci-kubernetes-test-go-release-1.6": {
+  "scenario": "kubernetes_verify",
+  "args": [
+    "--branch=release-1.6",
```

Comment:
In general explicit behavior is more robust and easier to understand. So the annoyance of adding an extra flag is probably less overall than writing the intelligence and then debugging the inevitable edge cases